### PR TITLE
[#7] Refactor docker compose to simplify running on Windows

### PR DESCRIPTION
**Pull request by kostobog**
- updating README.md

### CODE REVIEW

**Comment by blcham** [comment]
@kostobog @rodionnv what do you think about this configuration that reflects issue https://github.com/kbss-cvut/s-pipes-editor-ui/issues/23. Does it work for you as described in README.md now?

File: bin/spe.sh
Comment (blcham) [inline bin/spe.sh:1]: @rodionnv Please test if it works for you on Windows. If not either propose change or create new `.bat` file that would work for you (i would use just ChatGPT to rewrite the file)

File: README.md [hunk 1]
```diff
@@ -3,20 +3,38 @@
...
+- `$SHARED_ROOT` -- root folder that you want to share between docker services and your host system. It is the folder that
+  will be visible from docker services as well as from host filesytem and paths to all files will be the same.
+  Thus one can copy absolute path to a file from docker service and open it from host filesystem and other way around.
+  For linux the typical value would be `/home` or `/home/$user` and for windows the typical value would be `/host_mnt` or
+  `/host_mnt/c/Users/user1/code`.
+  
 ### Running Editor UI
 
+Default configuration assumes that:
+- you have **at least** docker-compose v2.20.2 (due to https://github.com/docker/cli/issues/4265; earlier versions were not tested)
```
Comment (blcham) [inline README.md:16]: @kostobog @rodionnv watch out this

**Review by rodionnv — COMMENTED** [review]
Doesn't work for me, I am not sure if I have correctrly configured my .env file
`SCRIPTPATHS=/c/D/ctu-fee/kbss/s-pipes/doc/examples
SCRIPTRULES=/c/D/ctu-fee/kbss/s-pipes-editor/src/main/resources/rules
RDF4J_REPOSITORYURL=http://rdf4j:8080/rdf4j-server/repositories
RDF4J_REPOSITORYNAME=s-pipes-hello-world
RDF4J_PCONFIGURL=/c/D/ctu-fee/kbss/s-pipes/doc/examples/hello-world/config.ttl
SHARED_ROOT=/host_mnt`
I have tried to configure it in different ways but I always get `s-pipes-editor-ui-s-pipes-engine-1  | java.nio.file.NoSuchFileException: /c/D/ctu-fee/kbss/s-pipes/doc/examples`

**Comment by blcham** [comment]
@rodionnv we did experiment with @kostobog and he did set up only:
`SHARED_ROOT=/host_mnt/c`
`PWD=/host_mnt/c/some-folder/s-pipes-editor-ui`

But there are 2 ways to execute it:
- inside wsl shell (this is probably better to do)
- inside windows shell (i guess this was the case for @kostobog)


@kostobog  please, share your experience so @rodionnv won't have to explore solutions

**Comment by blcham** [comment]
@rodionnv please share experience with running in PowerShell and WSL @rodionnv  + propose changes to README so everybody from Windows OS will understand

**Pull request merged**